### PR TITLE
Add error message for checking postgres install configuration

### PR DESCRIPTION
**Pull request by jingyimei**
MADlib needs to be installed on a postgres with python extension. If
postgres is not configured with the `--with-python` option, madpack
installation will failed for not finding $libdir/plpython2 directory.
This commit add some instructions when this failure happens.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/245/

File: src/madpack/madpack.py [hunk 1]
```diff
@@ -512,7 +512,9 @@ def _plpy_check(py_min_ver):
         try:
             _internal_run_query("CREATE LANGUAGE plpythonu;", True)
         except:
-            _error('Cannot create language plpythonu. Stopping installation...', False)
+            _error("""Cannot create language plpythonu. Please check if you
+                have configured and installed postgres with `--with-python`
```
Comment (edespino) [inline src/madpack/madpack.py]: Can you possibly use "portid" or "args.platform[0]" in place of "postgres"?

Comment (jingyimei) [reply src/madpack/madpack.py]: Sure, that should be more general.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/246/

**Comment by fmcquillan99** [comment]
Please replace "portid (your platform)" with actual values in the message.

**Pull request closed without merging**